src/app/app-interactive-map/app-interactive-map.component.ts

```diff
@@ -314,7 +317,10 @@ export class AppInteractiveMapComponent implements OnInit, AfterViewInit, OnDest
       }
       builder.set_knockout_reactions(card.knockoutReactions);
       builder.set_knockout_genes(card.knockoutGenes);
-      builder.set_added_reactions(card.addedReactions.map((reaction) => reaction.bigg_id));
+      builder.set_added_reactions(card.addedReactions
+        .filter((reaction) => !reaction.bigg_id.startsWith('DM_'))
+        .map((reaction) => reaction.bigg_id),
+      );
```

Comment:
we can not filter before added the reactions to the card? when we are putting the added reactions inside of the card??

Reply:
we can) now reactions are filtered before we add them to the card